### PR TITLE
Sample: Add hybrid speech sample

### LINKED ISSUE #1597: SAMPLE: Web Speech and Bing/Speech services text-to-speech combination
Hi,

In the v3 webchat it was possible to combine webspeech for speech to text and bing speech for text to speech in a one solution. 

I have tried to find a way how to do it in v4 but no luck so far.

Our basic requirement is to use 
- webspeech for speech recognition
- MS speech services or bing speech for text to speech synthesis
- also we have to be able to specify locale that is going to be used in both directions

I have looked the docs and samples but did not find any help

### PR DESCRIPTION
**Pull request by compulim**
> Fix #1597.

This PR add a sample of combining two speech engines:

- Speech-to-text: speech engine from the browser
- Text-to-speech: Cognitive Services Speech Services engine

### CODE REVIEW

**Comment by corinagum** [comment]
Please add README.md :) I can help with edits.

**Comment by compulim** [comment]
@corinagum added README.md and fixed comments.

File: samples/06.f.hybrid-speech/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+- To enable speech by using different engines:
+   - Speech-to-text: browser-supported speech engine
+   - Text-to-speech: Cognitive Services Speech Services engine
+
+We will start by using the [Cognitive Services Speech Services sample](./../06.c.cognitive-services-speech-services-js/README.md) as our Web Chat template.
+
+> Web browser speech package is available in the Web Chat core bundle and the full bundle, and you can use either CDN in your bot.
+
+[TBD]
```
[the remark this answers is not in the document]
Comment (compulim) [reply samples/06.f.hybrid-speech/README.md]: This README is not complete yet. But I believe we still want to push it out of the door first?

File: samples/06.f.hybrid-speech/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        // https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-direct-line-3-0-authentication
+
+        const directLineTokenRes = await fetch('https://webchat-mockbot.azurewebsites.net/directline/token', { method: 'POST' });
+        const { token } = await directLineTokenRes.json();
+
+        // For this demo, we are using our authorization token. For your production code, you should use a token from your Cognitive Services subscription.
+        const speechServicesTokenRes = await fetch('https://webchat-mockbot.azurewebsites.net/speechservices/token', { method: 'POST' });
+        const { region, token: authorizationToken } = await speechServicesTokenRes.json();
+
++       // We are creating a hybrid ponyfill factory that will merge the result of two ponyfill factories together.
```
[the remark this answers is not in the document]
Comment (compulim) [reply samples/06.f.hybrid-speech/README.md]: Let's do it!

File: samples/06.f.hybrid-speech/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (corinagum) [reply samples/06.f.hybrid-speech/README.md]: ah I see. Hmmm maybe just 
> Note: this sample README.md is not yet complete

instead of TBD?

**Comment by corinagum** [comment]
If you want to leave the remaining items for when I do sample clean up, that's fine. Could you leave a note on #1444?

File: samples/06.f.hybrid-speech/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (compulim) [reply samples/06.f.hybrid-speech/README.md]: Fixed `index.html` for this sample.

**Pull request merged**